### PR TITLE
chore: update ogmios to 5.1.0

### PR DESCRIPTION
**Pull request by rhyslbw**
# Context
I've split this commit out into a separate branch to merge in isolation. Ogmios 5.1.0 is out, and I'm targeting this version in my current WIP. There's a breaking change relating to Plutus scripts, and some numbers are typed as `bigint`

# Proposed Solution
Add the Plutus script type field to the GraphQL query, then use this to determine the Plutus version when converting to core types.

### CODE REVIEW

File: packages/cardano-graphql/src/WalletProvider/queryTransactions/graphqlTransactionsToCore.ts [hunk 1]
```diff
@@ -32,7 +32,7 @@ const scriptToCore = (script: GraphqlScript): Cardano.Script => {
     return { native: nativeScriptToCore(script) };
   } else if (script.__typename === 'PlutusScript') {
     return {
-      plutus: script.cborHex
+      'plutus:v1': script.cborHex
```
Comment (mkazlauskas) [inline packages/cardano-graphql/src/WalletProvider/queryTransactions/graphqlTransactionsToCore.ts:35]: Should probably do something like:
```
[script.type === 'PlutusScriptV1' ? 'plutus:v1' : 'plutus:v2']: script.cborHex
```

Comment (rhyslbw) [reply packages/cardano-graphql/src/WalletProvider/queryTransactions/graphqlTransactionsToCore.ts:35]: yeah I was looking for a field that would give me a classification, but there's no `.type` property on `script` or similar. Do you mean refactor to make this possible?

Comment (mkazlauskas) [reply packages/cardano-graphql/src/WalletProvider/queryTransactions/graphqlTransactionsToCore.ts:35]: No need to refactor, just add it to [query](https://github.com/input-output-hk/cardano-js-sdk/blob/master/packages/cardano-graphql/src/operations/transactions.graphql#L102-L105), then `yarn build && yarn generate` and it should be available

File: packages/cardano-graphql/src/WalletProvider/queryTransactions/graphqlTransactionsToCore.ts [hunk 2]
```diff
@@ -32,7 +32,7 @@ const scriptToCore = (script: GraphqlScript): Cardano.Script => {
     return { native: nativeScriptToCore(script) };
   } else if (script.__typename === 'PlutusScript') {
     return {
-      plutus: script.cborHex
+      [script.type === 'PlutusScriptV1' ? ('plutus:v1' as keyof Cardano.Script) : 'plutus:v2']: script.cborHex
```
Comment (mkazlauskas) [inline packages/cardano-graphql/src/WalletProvider/queryTransactions/graphqlTransactionsToCore.ts:35]: I'm wondering if `('plutus:v1' as const)` would do the trick here

**Review by mkazlauskas — APPROVED** [review]
Looks good!

File: packages/cardano-graphql/src/WalletProvider/queryTransactions/graphqlTransactionsToCore.ts (hunk 2 above)
Comment (rhyslbw) [reply packages/cardano-graphql/src/WalletProvider/queryTransactions/graphqlTransactionsToCore.ts:35]: image

**Pull request merged**